src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -918,7 +918,7 @@
   <Target Name="_RunWasmOptPostLink" Condition="'$(WasmRunWasmOpt)' == 'true'">
     <Error Condition="'$(_WasmOutputFileName)' == ''" Text="Could not determine %24(_WasmOutputFileName)" />
 
-    <Exec Command="wasm-opt$(_ExeExt) @(WasmOptConfigurationFlags, ' ') &quot;$(_WasmIntermediateOutputPath)$(_WasmOutputFileName)&quot; -o &quot;$(_WasmIntermediateOutputPath)$(_WasmOutputFileName)&quot;"
+    <Exec Command="wasm-opt$(_ExeExt) --enable-simd --enable-exception-handling --enable-bulk-memory @(WasmOptConfigurationFlags, ' ') &quot;$(_WasmIntermediateOutputPath)$(_WasmOutputFileName)&quot; -o &quot;$(_WasmIntermediateOutputPath)$(_WasmOutputFileName)&quot;"
```

Comment:
we do this by default now ?

Reply:
the enabled exception handling and simd should be fine here, it will not add any new instructions AFAIK, similar to what we already do in the wasm.proj.

the --enable-bulk-memory is now needed for some of the memory related calls we do

Reply:
we will want bulk memory anyway, it makes memset and memcpy faster

Reply:
> the enabled exception handling and simd should be fine here, it will not add any new instructions AFAIK, similar to what we already do in the src/mono/browser/runtime/CMakeLists.txt
> 
> the --enable-bulk-memory is now needed for some of the memory related calls we do

I guess we should verify that